File: euler/1-10/euler7.py
# By listing the first six prime numbers: 2, 3, 5, 7, 11, and 13, we can see that the 6th prime is 13.
# What is the 10,001st prime number?


def isPrime(number):
    for i in range (2, (number//2+1)):
        if number % i==0:
            return False 
    return True

a = int(input("-->"))

primes=[]
counter = 1
while len(primes) < a:
    counter +=1
    if isPrime(counter) == True:
        primes.append(counter)
        print(len(primes),":",counter)


print(primes)

File: euler/JS/euler7.js
// What is the 10001st prime numbe
const primeList=[2];
let i=2;

function isPrime (number, array) {
    for (let element of array){
      if (number % element === 0) {
        return false;
      }
    }
    return true;
   }

while (primeList.length<=10000) {
    if (isPrime(i,primeList)==true) {primeList.push(i);}
    i++;
}

console.log(primeList[10000]);